Tensile/KernelWriterAssembly.py

```diff
@@ -10995,6 +11012,11 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
         if edge and (beta or atomic):
           kStr += inst("s_mov_b{}".format(kernel["WavefrontSize"]), self.exec, -1, "full mask -1 -> exec" )
 
+    kStr += loadBetaCode
```

Comment:
Load beta codes are moved to here. The addrCalc.addrVgpr has been re-set for D matrix but not for C matrix.